server/src/environments/environment.ts

```diff
@@ -25,12 +25,12 @@ let envConfig = {
     KNOWLEDGEBASE_URL: process.env.KNOWLEDGEBASE_URL,
     KNOWLEDGEBASE_SECRET_KEY: process.env.KNOWLEDGEBASE_SECRET_KEY,
     TELEGRAM_TOKEN: process.env.TELEGRAM_TOKEN,
+    VIBER_TOKEN: process.env.VIBER_TOKEN,
     CONTAINER_VERSION: process.env.CONTAINER_VERSION,
     LOGGLY_TOKEN: process.env.LOGGLY_TOKEN,
     LOGGLY_SUBDOMAIN: process.env.LOGGLY_SUBDOMAIN ?? 'covid19liveupd',
     LOGGLY_TAGS: tags,
     APP_URL: process.env.APP_URL ?? '',
-    NGROK_URL: process.env.APP_URL ?? '',
```

Comment:
@danbilokha Do we stop using NGROK?

Reply:
I mean for local development

Reply:
nope, but I think it is misleading property since it has to be `APP_URL`